lug/config-production.yaml

```diff
@@ -0,0 +1,175 @@
+interval: 10 # Interval between pollings
```

Comment:
This file should be `config.yaml`. You do not need to create a separate file for production.